package.json

```
@@ -16,7 +16,13 @@
     "build:lv-web": "NEXT_PUBLIC_APP_SOURCE_NAME=lv-web turbo run build --filter=lv-web...",
     "build:caleuche": "NEXT_PUBLIC_APP_SOURCE_NAME=caleuche turbo run build --filter=caleuche...",
     "build:bagan": "NEXT_PUBLIC_APP_SOURCE_NAME=bagan turbo run build --filter=bagan...",
-    "ui": "npm run --filter=@repo/ui ui"
+    "ui": "npm run --filter=@repo/ui ui",
+    "test": "npm run test:jest && npm run test:pytest",
+    "test:jest": "npm run test --workspace=apps/lv-web",
```

```suggestion
    "test:jest": "npm run test --workspace=apps/lv-web",
    "test:watch": "npm run test:watch --workspace=apps/lv-web",
```
Add test:watch that hot reloads tests when you are editing frontend. It can be useful. Probably update readme for this

---

Next.js updated to 15.5.9 (new vulns. were found after 15.2.6, now shows 0 vulnerabilities) everywhere, and added change to root package.json to override other packages like next-auth - > Tested locally : Jest tests pass (7/7) and build succeeds with the new version.
Added .coverage and htmlcov/ to .gitignore
Added Windows/WSL commands to README for opening HTML reports
Added test:watch script to root package.json